diff --git a/Makefile.am b/Makefile.am
index c5b15ab..9f83379 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -5,14 +5,10 @@ bin_PROGRAMS = knock
 man_MANS = doc/knock.1
 
 if BUILD_KNOCKD
-sbin_PROGRAMS = knockd
-dist_sbin_SCRIPTS = src/knock_helper_ipt.sh
+bin_PROGRAMS += knockd
 man_MANS += doc/knockd.1
-sysconf_DATA = knockd.conf
 endif
 
-dist_doc_DATA = README.md TODO ChangeLog COPYING
-
 knock_SOURCES = src/knock.c
 knockd_SOURCES = src/knockd.c src/list.c src/list.h src/knock_helper_ipt.sh
 
