diff --git a/src/app.js b/src/app_fixed.js
index 5cdb5b1..dad4210 100644
--- a/src/app.js
+++ b/src/app_fixed.js
@@ -19,9 +19,14 @@ window.Worker = class Worker extends oldWorker {
         super(twitchBlobUrl);
 
         this.addEventListener("message", (event) => {
-            const { data } = event;
-            if ((data.id === 1 || isVariantA) && data.type === 1) {
-                this.postMessage({ ...data, arg: [data.arg] });
+            const data = event.data;
+
+            if ((data.id == 1 || isVariantA) && data.type == 1) {
+                const newData = event.data;
+
+                newData.arg = [data.arg];
+
+                this.postMessage(newData);
             }
         });
     }
